src/modules/mc_pos_control/mc_pos_control_main.cpp

```diff
@@ -1564,7 +1567,7 @@ MulticopterPositionControl::task_main()
 					if (!_takeoff_jumped) {
 						// ramp thrust setpoint up
 						if (_vel(2) > -(_params.tko_speed / 2.0f)) {
-							_takeoff_thrust_sp += 0.5f * dt;
+							_takeoff_thrust_sp += 1.0f * dt;
```

Comment:
I agree, but the flight test is not just needed because of this number but because the jumping is now actually enabled while beforehand we're not too sure. It might have been enabled if and only if the takeoff waypoint was set > 3m.